docker-compose.qemu.yml

```diff
@@ -1,8 +1,21 @@
 version: "2"
 
 services:
+  swtpm:
+    build:
+      context: ./swtpm
+      args:
+        - BALENA_ARCH=${BALENA_ARCH:-amd64}
```

Comment:
Don't need these build args anymore, can replace it with `build: swtpm`

Reply:
Good catch, thanks